andela/cyclops-backend
# #167728020 Password reset via email

**What does this PR do?**
- Resets user password via email

**Description of Task to be completed?**
Have the following Endpoints working
- POST `/api/v1/auth/forgotPassword`
- PUT `/api/v1//auth/resetPassword/:uuid/:token`

**How should this be manually tested?**
After cloning the repo,
- Create a SendGrid [account.](https://sendgrid.com/docs/)
- Create an API Key.
- Create an `.env` file and set `SENDGRID_API_KEY="The API key you created"`
- In that same file, make sure your database url is set as well
- Run migrations
- Run all seeds
- And it will be good to go

**Any background context you want to provide?**
To know more about how to send emails using sendGrid, go to the [Getting started with sendGrid API](https://sendgrid.com/docs/for-developers/sending-email/api-getting-started/)

**What are the relevant stories?**
- [#167728020](https://www.pivotaltracker.com/n/projects/2381984/stories/167728020)

**Screenshots (if appropriate)**
![Screenshot (87)](https://user-images.githubusercontent.com/29434750/63933162-99eab100-ca50-11e9-92f1-97638d32a5e4.png)

**Questions:**
- None
